src/Microsoft.ApplicationInsights.AspNet/Extensibility/Implementation/Tracing/TelemetryLoggerScope.cs

```diff
@@ -0,0 +1,37 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TelemetryLoggerScope.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Microsoft.ApplicationInsights.AspNet.Extensibility.Implementation.Tracing
+{
+    using System;
+
+    /// <summary>
+    /// Logging scope for <code>TelemetryLogger.BeginScopeImpl</code> calls.
+    /// </summary>
+    public class TelemetryLoggerScope : IDisposable
```

Comment:
We should discuss what we should do with this class since it isn't really even being used, there is a bigger ILogger design question here.